electron/src/weights_manifest.json

```diff
@@ -0,0 +1,74 @@
+[
```

Comment:
Downloading these weights from the internet for every time is not necessary, so save it as a json file